sentry-android-core/src/main/java/io/sentry/android/core/AndroidOptionsInitializer.java

```diff
@@ -115,6 +116,14 @@ private static void installDefaultIntegrations(
       final @NotNull IBuildInfoProvider buildInfoProvider,
       final @NotNull ILoadClass loadClass) {
 
+    options.addIntegration(
```

Comment:
👍  Just so that it doesn't get lost - I think we should expose ideally single method that turns on caching which takes `cachePathDir` as parameter and under the hood also turns on all cache-related integrations.